src/watcher.js

```diff
@@ -51,7 +54,14 @@ async function initialize() {
 async function runMain({ sendToQueue }) {
```

Comment:
Should `process.exit(1)` in the line 50 use a constant from `EXIT_CODES` as well?